lib/matplotlib/mpl-data/stylelib/seaborn-v0_8-dark.py

```diff
@@ -0,0 +1,16 @@
+import ast
+from pathlib import Path
+
+
+__mpl_style__ = {
+    **ast.literal_eval(Path(__file__).with_name("_seaborn-v0_8-common.py").read_text()),
```

Comment:
This is what I'm dreading about python style files: Hard-to-follow dynamically defined contents.

IMHO we should use algorithmic code structures very sparingly in style files. There's great value in a declarative definition.

Furthermore, I assume that that this is done to prevent repetition and enforce consistency (DRY). However, style files are static (in fact the only time we have changed these ever was renaming them once). So, getting accidentally inconsistent is not a practical problem.

Let's just keep the pattern from the .mplstyle file of writing everything out and marking the common and individual parts by comments.

Reply:
Can we not just make the common file normal Python and import it, instead of replicating all that with our own evaluation stuff?

Reply:
I'm not quite clear what you mean.

- We obviously have to continue to support `*.mplstyle` files
- We additionally want a (for now internal) `.py` based style file. We can freely decide the structure, but `__mpl_style__ = {...}` seems a reasonable choice. 

My concrete suggestion here is to not touch the seaborn .mplstyle files in this PR. It should only introduce `stylelib\xkcd.py` and the machinery to handle that.

Reply:
I mean, _if_ we want to change seaborn's styles, then instead of this `ast.literal_eval` work, it'd be simpler to give it an importable name, and do something like:
```python
from . import _seaborn08_common

__mpl_style__ = {
    **_seaborn08_common.__mpl_style__,
    ...,
}
```
But again, that's only if we want to change these files with inheritance in this manner.

Reply:
Using imports is tricky because style files currently live in mpl-data and thus not directly importable without playing tricks with the import machinery, which seems to be too complex to be worth it.
If the general agreement is to not change the seaborn styles at all I can just revert that part of the change.